packages/stringsifter/PKGBUILD

```diff
@@ -0,0 +1,57 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=stringsifter
+pkgver=14.f4331a0
+pkgrel=1
+pkgdesc='Machine learning tool that automatically ranks strings based on their relevance for malware analysis.'
+arch=('any')
+groups=('blackarch' 'blackarch-binary' 'blackarch-forensic')
+url='ihttps://github.com/fireeye/stringsifter'
+license=('Apache')
+depends=('python' 'python-virtualenv' 'python-numpy' 'python-scikit-learn' \
+  'python-joblib' 'python-pytest')
```

Comment:
`python-pytest` is only for the tests that we remove and don't package so remove this dependency

Reply:
removed.